zulipterminal/model.py

```diff
@@ -119,6 +119,17 @@ def __init__(self, controller: Any) -> None:
         self.new_user_input = True
         self._start_presence_updates()
 
+    def get_versions(self) -> Dict[str, str]:
```

Comment:
We shouldn't need to explicitly fetch this information. You can check the new https://chat.zulip.org/api/changelog, or `git checkout` various versions of the source, but this can be requested to be (and later will be included by default) in the initial data sent by the server.